### PR TITLE
Callout creation dialog

### PR DESCRIPTION
**Pull request by hero101**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

File: src/components/composite/aspect/AspectCreationDialog/form/CalloutForm.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    onChange?.(callout);
+  };
+
+  const helpText = useMemo(() => {
+    if (callout?.type === CalloutType.Card) {
+      return t('components.callout-creation.info-step.type-cards-help');
+    } else if (callout?.type === CalloutType.Canvas) {
+      return t('components.callout-creation.info-step.type-canvases-help');
+    }
+    if (callout?.type === CalloutType.Discussion) {
```
Comment (ccanos) [inline src/components/composite/aspect/AspectCreationDialog/form/CalloutForm.tsx:81]: This code works as is, but why not an `else if` here, or a `switch` ?

File: src/components/composite/aspect/AspectCreationDialog/form/CalloutForm.tsx [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+      return t('components.callout-creation.info-step.type-comments-help');
+    } else {
+      return '';
+    }
+  }, [callout]);
+
+  const calloutTypes = [
+    { id: CalloutType.Card, name: 'Cards' },
```
Comment (ccanos) [inline src/components/composite/aspect/AspectCreationDialog/form/CalloutForm.tsx:89]: I think these should be `t('....')`

File: src/domain/callout/creation-dialog/step-layout/StepLayout.tsx [hunk 3]
```diff
@@ -0,0 +1,130 @@
+import React, { FC, useMemo } from 'react';
```
Comment (ccanos) [inline src/domain/callout/creation-dialog/step-layout/StepLayout.tsx:1]: I think this whole file should be in a common place for it to be used by other wizards/stepped forms.
In fact I think we have already something

Comment (hero101) [reply src/domain/callout/creation-dialog/step-layout/StepLayout.tsx:1]: It's a layout used for creating callouts. I don't expect it to be used from the outside.

**Review by ccanos — APPROVED** [review]

**Pull request merged**